frontend/src/types/SelectInputTypes.ts

```diff
@@ -0,0 +1,10 @@
+import { OptionBase } from "chakra-react-select";
+
+export interface Option extends OptionBase {
+  value: string | boolean;
+  label: string;
+}
+
+export interface MultiOption extends Option {
+  value: string;
+}
```

Comment:
why do we need to define this extra `MultiOption` interface?

Reply:
Since multi select inputs wouldn't have a boolean as a possible value, I thought the best approach would be to extend an interface for that specific use case

Reply:
Another example is in `SelectFormInputClassroom`, where the Option defined in `SelectInputTypes` wouldn't match the type, but `MultiOption` does.